components/annotationList.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import Router from "next/router";
+import { render } from "react-dom";
+
+/**
+ * annotations(annArray) : function that returns 1 array of all the annotation tags
+ *
+ * @param {*} annArray : array of annotations (which each annotation contains an array of tags)
+ */
+function annotations(annArray) {
+  if (Array.isArray(annArray)) {
```

Comment:
I think it is more appropriate to do
`  if (annArray.length === 0) return;` 
And then get rid of the else. What do you think?

Reply:
or maybe return message telling there is noting to display